robot/Cumulus/resources/ContactPageObject.py

```diff
@@ -35,7 +35,8 @@ def update_field_value(self,field_name,old_value,new_value):
     def go_to_related_engagement_actionplans_page(self,customerid):
 
         """ Navigates to the related engagement plans listing page"""
-        objectname = "Action_Plans__r"
+        name = "Action_Plans__r"
+        objectname = "{}{}".format(self.npsp.get_npsp_namespace_prefix(), name)
```

Comment:
What's the difference between `self.npsp.get_npsp_namespace_prefix()` and the `get_namespace_prefix() or get_namespace_prefix()` formula used elsewhere?

Reply:
get_namespace_prefix() keyword I had to pass package name to get the namespace prefix and with 2GP now we have 2 names. get_npsp_namespace_prefix()(this is created by David Glick long time back) gets me the namespace in both cases but when I am calling NPSP pageobject from Connector repo it was not returning namespace prefix.